src/sass/_BrandIcon.scss

```diff
@@ -22,87 +23,87 @@
 // a future release. Until then, they are mapped to 'one' and 'xlsx'.
 
 .ms-BrandIcon--onepkg.ms-BrandIcon--icon16 {
-  background-image: url(https://spoprod-a.akamaihd.net/files/fabric/assets/item-types-fluent/16/one.png);
+  background-image: url('#{$ms-cdn-path}/item-types-fluent/16/one.png');
```

Comment:
As long as you're changing these, you should be able to replace `item-types-fluent` with `item-types`. That's the more up to date and maintained assets directory.

Reply:
will make the change!